docs/nddata/subclassing.rst

```diff
@@ -58,9 +58,9 @@ Customize the setter for a property
     ...         self._mask = np.array(value, dtype=np.bool_)
 
     >>> ndd = NDDataMaskBoolNumpy([1,2,3])
-    >>> ndd.mask = True
+    >>> ndd.mask = True, False, True
```

Comment:
I know this is valid, but maybe we want to make it `[True, False, True]` to be more explicit? This might look strange to new users.